### PR TITLE
feat: added timestamp validity check in get_metric_range_data

### PR DESCRIPTION
**Pull request by anushkrishnav**
Fixes Issue #191 
Made the require addition and test

### CODE REVIEW

File: tests/test_prometheus_connect.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def test_get_metric_range_data(self):  # noqa D102
             < metric_objects_list[0].end_time.timestamp(),
             "invalid metric end time",
         )
+        self.assertFalse(end_time < start_time, "invalid metric end time")
```
Comment (4n4nd) [inline tests/test_prometheus_connect.py:55]: Can you please write a test case where you use the invalid start_time and end_time in the function that you updated?
Something like this: https://github.com/AICoE/prometheus-api-client-python/blob/559434db91bd59af7adadc41456a3f889f09ddb6/tests/test_prometheus_connect.py#L91-L96

File: prometheus_api_client/prometheus_connect.py [hunk 2]
```diff
@@ -196,6 +196,9 @@ def get_metric_range_data(
         if not (isinstance(start_time, datetime) and isinstance(end_time, datetime)):
             raise TypeError("start_time and end_time can only be of type datetime.datetime")
 
+        if end_time < start_time:
```
Comment (4n4nd) [inline prometheus_api_client/prometheus_connect.py:199]: Can you also add this check in all other range query functions which utilize the start_time and end_time parameters?

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: prometheus_api_client/prometheus_connect.py (hunk 2 above)
Comment (anushkrishnav) [reply prometheus_api_client/prometheus_connect.py:199]: Suree

**Comment by anushkrishnav** [comment]
I will re open the PR need to add few more test

**Pull request closed without merging**